src/styles/styles.js

```diff
@@ -783,6 +783,15 @@ const styles = {
         justifyContent: 'center',
     },
 
+    chatItemTopPadding: {
+        backgroundColor: 'rgba(0,0,0,0)',
```

Comment:
Why are we adding a transparent background here?

Reply:
Here are the results if I remove that line (787) of code
![Screenshot 2021-01-08 at 11 08 23 PM](https://user-images.githubusercontent.com/4520893/104064555-d0b90580-5206-11eb-9b29-1b48b47790f2.png)

Reply:
seems the typing area has it's own background independent of it's parent

Reply:
Why is the typing area getting pushed outside of the containing view?

Reply:
because of this entry `top: 10,`

Reply:
> Thanks for these changes. Unfortunately, they are breaking styles on web/desktop platform. Please be sure to **test all supported platforms** before submitting this PR again.
> 
> **Web/Desktop:**
> ![2020-12-29_08-54-33](https://user-images.githubusercontent.com/32969087/103307490-470e6a00-49b4-11eb-8ec0-f3a42fce158f.png)
> ![2020-12-29_08-54-41](https://user-images.githubusercontent.com/32969087/103307493-4970c400-49b4-11eb-85ba-b9cd063180c6.png)
> 
> **iOS:**
> ![2020-12-29_09-10-25](https://user-images.githubusercontent.com/32969087/103308417-80e07000-49b6-11eb-8adc-2361c9dee372.png)

Kindly note that to fix the above issue `position: 'relative'` did the tricky.